.github/workflows/ci-build.yml

```diff
@@ -2,26 +2,31 @@ name: CI Build
 on: pull_request
 
 jobs:
-  build:
+  check-lockfile:
```

Comment:
Removed the deploy job from this file, copied check-lockfile and check-req-json jobs from `ci-check.yml`